src/extension/android/adb.ts

```diff
@@ -41,14 +45,23 @@ export interface IDevice {
 const AndroidSDKEmulatorPattern = /^emulator-\d{1,5}$/;
 
 export class AdbHelper {
-    private static childProcess: ChildProcess = new ChildProcess();
-    private static commandExecutor: CommandExecutor = new CommandExecutor();
+    private childProcess: ChildProcess = new ChildProcess();
+    private commandExecutor: CommandExecutor = new CommandExecutor();
+    private adbExecutable: string = "";
+
+    constructor(projectRoot: string, logger: ILogger) {
```

Comment:
Can we make `logger` optional?

Reply:
@iTOYS yes, we can indeed, thanks for pointing to this!